lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -707,7 +711,8 @@ def _rescale_shape_parameters(self, mean, std):
                 raise ValueError(msg)
 
 
-class GeneratePercentilesFromMeanAndVariance(BasePlugin, FromMeanAndVariance):
+class ConvertLocationAndScaleParametersToPercentiles(
+        BasePlugin, ConvertLocationAndScaleParameters):
```

Comment:
The process function, and the _location_and_scal_parameters_to_percentiles both have parameters that are described as means and variances. Does this need to be fixed too?

Reply:
I've hopefully corrected the variables names within this plugin.